diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index d59ba34..203696e 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -10,93 +10,91 @@
 
 'use strict';
 
+import typeof * as TurboModuleRegistry from './Libraries/TurboModule/TurboModuleRegistry';
+
 // Components
 import typeof AccessibilityInfo from './Libraries/Components/AccessibilityInfo/AccessibilityInfo';
-import typeof ActivityIndicator from './Libraries/Components/ActivityIndicator/ActivityIndicator';
-import typeof Button from './Libraries/Components/Button';
-import typeof DatePickerIOS from './Libraries/Components/DatePicker/DatePickerIOS';
-import typeof DrawerLayoutAndroid from './Libraries/Components/DrawerAndroid/DrawerLayoutAndroid';
-import typeof FlatList from './Libraries/Lists/FlatList';
-import typeof Image from './Libraries/Image/Image';
-import typeof ImageBackground from './Libraries/Image/ImageBackground';
-import typeof InputAccessoryView from './Libraries/Components/TextInput/InputAccessoryView';
-import typeof KeyboardAvoidingView from './Libraries/Components/Keyboard/KeyboardAvoidingView';
-import typeof MaskedViewIOS from './Libraries/Components/MaskedView/MaskedViewIOS';
-import typeof Modal from './Libraries/Modal/Modal';
-import typeof Pressable from './Libraries/Components/Pressable/Pressable';
-import typeof ProgressBarAndroid from './Libraries/Components/ProgressBarAndroid/ProgressBarAndroid';
-import typeof ProgressViewIOS from './Libraries/Components/ProgressViewIOS/ProgressViewIOS';
-import typeof RefreshControl from './Libraries/Components/RefreshControl/RefreshControl';
-import typeof SafeAreaView from './Libraries/Components/SafeAreaView/SafeAreaView';
-import typeof ScrollView from './Libraries/Components/ScrollView/ScrollView';
-import typeof SectionList from './Libraries/Lists/SectionList';
-import typeof Slider from './Libraries/Components/Slider/Slider';
-import typeof StatusBar from './Libraries/Components/StatusBar/StatusBar';
-import typeof Switch from './Libraries/Components/Switch/Switch';
-import typeof Text from './Libraries/Text/Text';
-import typeof TextInput from './Libraries/Components/TextInput/TextInput';
-import typeof Touchable from './Libraries/Components/Touchable/Touchable';
-import typeof TouchableHighlight from './Libraries/Components/Touchable/TouchableHighlight';
-import typeof TouchableNativeFeedback from './Libraries/Components/Touchable/TouchableNativeFeedback';
-import typeof TouchableOpacity from './Libraries/Components/Touchable/TouchableOpacity';
-import typeof TouchableWithoutFeedback from './Libraries/Components/Touchable/TouchableWithoutFeedback';
-import typeof View from './Libraries/Components/View/View';
-import typeof VirtualizedList from './Libraries/Lists/VirtualizedList';
-import typeof VirtualizedSectionList from './Libraries/Lists/VirtualizedSectionList';
-
 // APIs
 import typeof ActionSheetIOS from './Libraries/ActionSheetIOS/ActionSheetIOS';
+import typeof ActivityIndicator from './Libraries/Components/ActivityIndicator/ActivityIndicator';
 import typeof Alert from './Libraries/Alert/Alert';
 import typeof Animated from './Libraries/Animated/Animated';
-import typeof Appearance from './Libraries/Utilities/Appearance';
 import typeof AppRegistry from './Libraries/ReactNative/AppRegistry';
 import typeof AppState from './Libraries/AppState/AppState';
+import typeof Appearance from './Libraries/Utilities/Appearance';
 import typeof AsyncStorage from './Libraries/Storage/AsyncStorage';
 import typeof BackHandler from './Libraries/Utilities/BackHandler';
+import typeof Button from './Libraries/Components/Button';
 import typeof Clipboard from './Libraries/Components/Clipboard/Clipboard';
-import typeof DeviceInfo from './Libraries/Utilities/DeviceInfo';
+import typeof DatePickerIOS from './Libraries/Components/DatePicker/DatePickerIOS';
 import typeof DevSettings from './Libraries/Utilities/DevSettings';
+import typeof DeviceInfo from './Libraries/Utilities/DeviceInfo';
 import typeof Dimensions from './Libraries/Utilities/Dimensions';
+import typeof DrawerLayoutAndroid from './Libraries/Components/DrawerAndroid/DrawerLayoutAndroid';
+// Plugins
+import typeof {DynamicColorIOS} from './Libraries/StyleSheet/PlatformColorValueTypesIOS';
 import typeof Easing from './Libraries/Animated/Easing';
-import typeof ReactNative from './Libraries/Renderer/shims/ReactNative';
+import typeof FlatList from './Libraries/Lists/FlatList';
 import typeof I18nManager from './Libraries/ReactNative/I18nManager';
+import typeof Image from './Libraries/Image/Image';
+import typeof ImageBackground from './Libraries/Image/ImageBackground';
 import typeof ImagePickerIOS from './Libraries/Image/ImagePickerIOS';
+import typeof InputAccessoryView from './Libraries/Components/TextInput/InputAccessoryView';
 import typeof InteractionManager from './Libraries/Interaction/InteractionManager';
 import typeof Keyboard from './Libraries/Components/Keyboard/Keyboard';
+import typeof KeyboardAvoidingView from './Libraries/Components/Keyboard/KeyboardAvoidingView';
 import typeof LayoutAnimation from './Libraries/LayoutAnimation/LayoutAnimation';
 import typeof Linking from './Libraries/Linking/Linking';
 import typeof LogBox from './Libraries/LogBox/LogBox';
+import typeof MaskedViewIOS from './Libraries/Components/MaskedView/MaskedViewIOS';
+import typeof Modal from './Libraries/Modal/Modal';
 import typeof NativeDialogManagerAndroid from './Libraries/NativeModules/specs/NativeDialogManagerAndroid';
 import typeof NativeEventEmitter from './Libraries/EventEmitter/NativeEventEmitter';
+import typeof NativeModules from './Libraries/BatchedBridge/NativeModules';
 import typeof Networking from './Libraries/Network/RCTNetworking';
 import typeof PanResponder from './Libraries/Interaction/PanResponder';
 import typeof PermissionsAndroid from './Libraries/PermissionsAndroid/PermissionsAndroid';
 import typeof PixelRatio from './Libraries/Utilities/PixelRatio';
+import typeof Platform from './Libraries/Utilities/Platform';
+import typeof {PlatformColor} from './Libraries/StyleSheet/PlatformColorValueTypes';
+import typeof Pressable from './Libraries/Components/Pressable/Pressable';
+import typeof ProgressBarAndroid from './Libraries/Components/ProgressBarAndroid/ProgressBarAndroid';
+import typeof ProgressViewIOS from './Libraries/Components/ProgressViewIOS/ProgressViewIOS';
 import typeof PushNotificationIOS from './Libraries/PushNotificationIOS/PushNotificationIOS';
+import typeof RCTDeviceEventEmitter from './Libraries/EventEmitter/RCTDeviceEventEmitter';
+import typeof RCTNativeAppEventEmitter from './Libraries/EventEmitter/RCTNativeAppEventEmitter';
+import typeof ReactNative from './Libraries/Renderer/shims/ReactNative';
+import typeof RefreshControl from './Libraries/Components/RefreshControl/RefreshControl';
+import typeof {RootTagContext} from './Libraries/ReactNative/RootTag';
+import typeof SafeAreaView from './Libraries/Components/SafeAreaView/SafeAreaView';
+import typeof ScrollView from './Libraries/Components/ScrollView/ScrollView';
+import typeof SectionList from './Libraries/Lists/SectionList';
 import typeof Settings from './Libraries/Settings/Settings';
 import typeof Share from './Libraries/Share/Share';
+import typeof Slider from './Libraries/Components/Slider/Slider';
+import typeof StatusBar from './Libraries/Components/StatusBar/StatusBar';
 import typeof StyleSheet from './Libraries/StyleSheet/StyleSheet';
+import typeof Switch from './Libraries/Components/Switch/Switch';
 import typeof Systrace from './Libraries/Performance/Systrace';
+import typeof Text from './Libraries/Text/Text';
+import typeof TextInput from './Libraries/Components/TextInput/TextInput';
 import typeof ToastAndroid from './Libraries/Components/ToastAndroid/ToastAndroid';
-import typeof * as TurboModuleRegistry from './Libraries/TurboModule/TurboModuleRegistry';
+import typeof Touchable from './Libraries/Components/Touchable/Touchable';
+import typeof TouchableHighlight from './Libraries/Components/Touchable/TouchableHighlight';
+import typeof TouchableNativeFeedback from './Libraries/Components/Touchable/TouchableNativeFeedback';
+import typeof TouchableOpacity from './Libraries/Components/Touchable/TouchableOpacity';
+import typeof TouchableWithoutFeedback from './Libraries/Components/Touchable/TouchableWithoutFeedback';
 import typeof UIManager from './Libraries/ReactNative/UIManager';
-import typeof useColorScheme from './Libraries/Utilities/useColorScheme';
-import typeof useWindowDimensions from './Libraries/Utilities/useWindowDimensions';
 import typeof UTFSequence from './Libraries/UTFSequence';
 import typeof Vibration from './Libraries/Vibration/Vibration';
+import typeof View from './Libraries/Components/View/View';
+import typeof VirtualizedList from './Libraries/Lists/VirtualizedList';
+import typeof VirtualizedSectionList from './Libraries/Lists/VirtualizedSectionList';
 import typeof YellowBox from './Libraries/YellowBox/YellowBoxDeprecated';
-
-// Plugins
-import typeof {DynamicColorIOS} from './Libraries/StyleSheet/PlatformColorValueTypesIOS';
-import typeof NativeModules from './Libraries/BatchedBridge/NativeModules';
-import typeof Platform from './Libraries/Utilities/Platform';
-import typeof {PlatformColor} from './Libraries/StyleSheet/PlatformColorValueTypes';
-import typeof processColor from './Libraries/StyleSheet/processColor';
-import typeof RCTDeviceEventEmitter from './Libraries/EventEmitter/RCTDeviceEventEmitter';
-import typeof RCTNativeAppEventEmitter from './Libraries/EventEmitter/RCTNativeAppEventEmitter';
-import typeof {RootTagContext} from './Libraries/ReactNative/RootTag';
-
 import type {HostComponent as _HostComponentInternal} from './Libraries/Renderer/shims/ReactNativeTypes';
+import typeof processColor from './Libraries/StyleSheet/processColor';
+import typeof useColorScheme from './Libraries/Utilities/useColorScheme';
+import typeof useWindowDimensions from './Libraries/Utilities/useWindowDimensions';
 
 export type HostComponent<T> = _HostComponentInternal<T>;
 
@@ -435,32 +433,16 @@ module.exports = {
   },
   // Deprecated Prop Types
   get ColorPropType(): $FlowFixMe {
-    invariant(
-      false,
-      'ColorPropType has been removed from React Native. Migrate to ' +
-        "ColorPropType exported from 'deprecated-react-native-prop-types'.",
-    );
-  },
+    return require('deprecated-react-native-prop-types').ColorPropType;
+   },
   get EdgeInsetsPropType(): $FlowFixMe {
-    invariant(
-      false,
-      'EdgeInsetsPropType has been removed from React Native. Migrate to ' +
-        "EdgeInsetsPropType exported from 'deprecated-react-native-prop-types'.",
-    );
+    return require('deprecated-react-native-prop-types').EdgeInsetsPropType;
   },
   get PointPropType(): $FlowFixMe {
-    invariant(
-      false,
-      'PointPropType has been removed from React Native. Migrate to ' +
-        "PointPropType exported from 'deprecated-react-native-prop-types'.",
-    );
+    return require('deprecated-react-native-prop-types').PointPropType;
   },
   get ViewPropTypes(): $FlowFixMe {
-    invariant(
-      false,
-      'ViewPropTypes has been removed from React Native. Migrate to ' +
-        "ViewPropTypes exported from 'deprecated-react-native-prop-types'.",
-    );
+    return require('deprecated-react-native-prop-types').ViewPropTypes;
   },
 };
 
diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
new file mode 100644
index 0000000..21a13cf
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.env
@@ -0,0 +1 @@
+export RCT_METRO_PORT=8081
\ No newline at end of file
